package.json

```diff
@@ -22,9 +22,12 @@
     "prettier": "node prettier.js",
     "clean": "lerna clean --yes && rimraf .jest",
     "publish": "git pull --rebase && check-installed-dependencies && npm run build && lerna publish",
-    "update-snapshot": "cross-env BABEL_ENV=commonjs jest --no-cache --updateSnapshot"
+    "update-snapshot": "cross-env BABEL_ENV=commonjs jest --no-cache --updateSnapshot",
+    "styleguide": "styleguidist server"
   },
   "devDependencies": {
+    "@webpack-blocks/babel6": "^0.4.1",
+    "@webpack-blocks/webpack2": "^0.4.0",
```

Comment:
Found it in the docs for react-styleguidist https://github.com/styleguidist/react-styleguidist/blob/master/docs/Webpack.md#non-webpack-projects